mappings/net/minecraft/entity/mob/GhastEntity.mapping

```diff
@@ -10,6 +10,8 @@ CLASS net/minecraft/class_1571 net/minecraft/entity/mob/GhastEntity
 		ARG 3 pos
 		ARG 4 random
 	METHOD method_26913 createGhastAttributes ()Lnet/minecraft/class_5132$class_5133;
+	METHOD method_44320 isFireballFromPlayer (Lnet/minecraft/class_1282;)Z
```

Comment:
```suggestion
	METHOD method_44320 isFireballFromPlayer (Lnet/minecraft/class_1282;)Z
		COMMENT {@return whether {@code damageSource} is caused by a player's fireball}
		COMMENT
		COMMENT <p>This returns {@code true} for ghast fireballs reflected by a player,
		COMMENT since the attacker is set as the player in that case.
```

`isReflectedFireball` could work, but this also returns `true` if a mod allowed a player to shoot ghast fireballs, so I decided not to.